packages/mui-material/src/Select/Select.spec.tsx

```diff
@@ -45,3 +45,54 @@ function genericValueTest() {
   // disabledUnderline prop should be available (inherited from InputProps) and NOT throw typescript error
   <Select disableUnderline />;
 }
+
+function App() {
+  enum MyEnum {
+    FIRST = 'first',
+    SECOND = 'second',
+  }
+
+  const [selectedValue, setSelectedValue] = React.useState<MyEnum | ''>('');
+
+  return (
+    <React.Fragment>
+      {/* displayEmpty is true */}
+      <Select
+        renderValue={(value) => {
+          if (value === '') {
+            return 'None selected';
+          }
+          return value;
+        }}
+        displayEmpty
+        value={selectedValue}
+        onChange={(e) => setSelectedValue(e.target.value as MyEnum)}
+      >
+        <MenuItem value="">
+          <em>Blank</em>
+        </MenuItem>
+        <MenuItem value="first">first</MenuItem>
+        <MenuItem value="second">second</MenuItem>
+      </Select>
+
+      {/* displayEmpty is false */}
+      <Select
```

Comment:
Could you please also verify the `displayEmpty: true; multiple: true` case here?

Reply:
It's already covered in a demo, but still added it in spec here now.